src/olympia/constants/base.py

```diff
@@ -134,7 +134,7 @@
     _ADDON_LPADDON: _('Deprecated Language Pack (Add-on)'),
     _ADDON_PLUGIN: _('Deprecated Plugin'),
     _ADDON_PERSONA: _('Deprecated LWT'),
-    ADDON_STATICTHEME: _('Theme (Static)'),
+    ADDON_STATICTHEME: _('Theme'),
```

Comment:
These are also unrelated changes too, though coming way past time!

Reply:
Yes, goes with the drive-by fix mentioned above, mainly to avoid saying `'The type (Extension) does not match the type of your add-on on AMO (Theme (Static))` for instance, which looks weird.